src/foundry/common/documents.mjs/baseActor.d.ts

```diff
@@ -22,4 +25,41 @@ export declare class BaseActor extends Document<any, null> {
       types: string[];
     }
   >;
+  /*
+   * A reference to the Collection of embedded ActiveEffect instances in the Actor document, indexed by _id.
+   */
+  get effects(): this['data']['effects'];
+
+  /**
+   * A reference to the Collection of embedded Item instances in the Actor document, indexed by _id.
+   */
+  get items(): this['data']['items'];
+
+  /**
+   * Migrate the system data object to conform to data model defined by the current system version.
+   * @see mergeObject
+   * @param options - Options which customize how the system data is migrated.
+   * @returns The migrated system data object, not yet saved to the database
+   */
+  migrateSystemData(options?: MigrateSystemDataOptions): object;
+}
```

Comment:
`_preCreate` and `_preUpdate` are missing.

Reply:
Those two methods are inherited from Document, and even in Foundry they're not referenced in the BaseActor class

Reply:
They are overridden in BaseActor, at least that’s what I am seeing… are we working with different versions of foundry?

Reply:
It is overriden, but I figured I don't really need to include it since there's no changes to the docstring. so for the purpose of typing it makes no real difference. I can add them though, no problem

Reply:
We add all the things that are present in foundry, just for consistency :D. We even type functions that are neither available globally, nor exported :D